src/main/java/roman/RomanToNumberToRoman.java

```diff
@@ -46,4 +36,65 @@ private int getIntValueOfCurrentRoman(char currentRoman) {
 		}
 		return currentNumber;
 	}
-}
+
+	public String toRoman(int number) {
+		StringBuilder sb = new StringBuilder();
+		char currentNumeral = ' ';
+		int powerOf = 0;
+		int baseNumber = 0;
+		int numberOfRomans = 0;
+		while (number > 9) {
+			powerOf = (int) (Math.log10(number));
+			baseNumber = (int) (Math.pow(10, powerOf));
+			int newBaseNumber = 0;
+			numberOfRomans = number / baseNumber;
```

Comment:
Can you explain what `numberOfRomans` is tracking? Is it the number of multiples of 1, 10, 100, or 1000 ?